django_ipam/base/models.py

```diff
@@ -0,0 +1,35 @@
+import swapper
+from django.db import models
+from openwisp_utils.base import TimeStampedEditableModel
+
+
+class BaseModel(TimeStampedEditableModel):
+    id = None
```

Comment:
why this? Remove this `BaseModel` totally, we don't need it. We also need id fields to be `uuid`